python/ml4ir/applications/ranking/model/ranking_model.py

```diff
@@ -280,6 +281,19 @@ def mask_padded_records(predictions, features_dict):
             pad_sequence=pad_sequence,
         )
 
+        # Logging positional biases
+        for layer in self.model.layers:
+            if layer.name == PositionalBiasHandler.FIXED_ADDITIVE_POSITIONAL_BIAS:
+                positional_bias_coefficients = pd.DataFrame(
```

Comment:
@mohazahran Can you post an example here for how it would log?

Reply:
@lastmansleeping You mean an example of output log for positional bias?

Reply:
Yep, I saw that you added it. Thanks!